data/gegno/gegno power.txt

```diff
@@ -0,0 +1,134 @@
+	plural "R01 Skirmish Batteries"
+	cost 17500
+	mass 12
+	thumbnail outfit/gbatterytiny
+	"energy capacity" 1121
+	"outfit space" -12
+	description "Skirmish batteries are very common amongst ships across Gegno space. They can be either a perfectly convenient storage of battery power to offset the load of larger Gegno generators, or a nice addition to add on top of them where more energy would be needed."
+
+outfit "R02 Battlezone Battery"
+	category Power
+	licenses
+		"Gegno Civilian"
+	plural "R02 Battlezone Batteries"
+	cost 32000
+	mass 23
+	thumbnail outfit/gbatterysmall
+	"energy capacity" 2237
+	"outfit space" -23
+	description "Battlezone batteries are primarily used on the Gegno's medium-range combat vessels, holding a decent amount of power for long lasting engagements. They also fit nicely on larger vessels as an extra power source to feed their larger systems."
+
+outfit "R03 Warforge Battery"
+	category Power
+	licenses
+		"Gegno Civilian"
+	plural "R03 Warforge Batteries"
+	cost 61700
+	mass 41
+	thumbnail outfit/gbatterymed
+	"energy capacity" 4469
+	"outfit space" -41
+	description "Warforge batteries have proved their usefulness over the years as large, sturdy battery packs. They support the equally large volleys that are fired from Gegno ships, and help maintain an excess of power for engine usage."
+
+outfit "R04 Crusade Battery"
+	category Power
+	licenses
+		"Gegno Civilian"
+		"Vi Evocati"
+		"Vi Centurion"
+		"Vi Lord"
+	plural "R04 Crusade Batteries"
+	cost 136000
+	mass 83
+	thumbnail outfit/gbatterylarge
+	"energy capacity" 8885
+	"outfit space" -83
+	description "As the largest battery mounted on Gegno ships, the Crusade Battery supplies almost as much power as a small generator, enough to support the massive warships of both the Scin and the Vi. However, the Gegno are inefficient at increasing density with size, as done with human batteries, so these fall behind similarly sized options."
+
+outfit "C3 Scrap Cell"
+	category Power
+	licenses
+		"Gegno Civilian"
+	plural "C3 Scrap Cells"
+	cost 54000
+	mass 25
+	thumbnail outfit/vicoretiny
+	"energy generation" 1.4
+	"heat generation" 2.2
+	"outfit space" -25
+	description "The Scrap Cell is a small radiothermal generator that produces energy generation in a pinch, but its heat production can affect the Gegno's slightly hotter running tech. Its output is very limited, and is only used in tandem with the Gegno's smaller-sized systems."
+
+outfit "C7 Brawl Cell"
+	category Power
+	licenses
+		"Gegno Civilian"
+	plural "C7 Brawl Cells"
+	cost 68100
+	mass 39
+	thumbnail outfit/vicoresmall
+	"energy generation" 2.5
+	"heat generation" 3.9
+	"outfit space" -39
+	description "C7 Brawl Cells are a slight deviation from the other C class cores as it is set upright-forward, a more rudimentary configuration. The tried and tested design, however, still serves the Gegno well, though it will likely be phased out when they are better able to scale up newer technologies."
```

Comment:
What newer technologies?
```suggestion
	description "Unlike other C-class cores, C7 Brawl Cells are mounted horizontally. This design, however, still serves the Gegno well, though it will likely be phased out when they are better able to scale up newer technologies."
```

Reply:
Hm. Now I'm not so sure, lol. Maybe the idea was to scale *down* newer technologies, being newer upright generators.

Reply:
Revisited.